bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/MyRenaultHttpSession.java

```diff
@@ -233,108 +248,110 @@ public void getLocation(Car car)
+    public void getLockStatus(Car car)
+            throws RenaultForbiddenException, RenaultUpdateException, RenaultNotImplementedException {
+        JsonObject responseJson = getKamereonResponse("/commerce/v1/accounts/" + kamereonaccountId
+                + "/kamereon/kca/car-adapter/v1/cars/" + config.vin + "/lock-status?country=" + getCountry(config));
+        if (responseJson != null) {
+            car.setLockStatus(responseJson);
         }
     }
 
-    public void actionChargeMode(ChargingMode mode) throws RenaultForbiddenException, RenaultNotImplementedException {
-        Request request = httpClient
-                .newRequest(this.constants.getKamereonRootUrl() + "/commerce/v1/accounts/" + kamereonaccountId
-                        + "/kamereon/kca/car-adapter/v1/cars/" + config.vin + "/actions/charge-mode?country="
-                        + getCountry(config))
-                .method(HttpMethod.POST).header("Content-type", "application/vnd.api+json")
-                .header("apikey", this.config.kamereonApiKey)
-                .header("x-kamereon-authorization", "Bearer " + kamereonToken).header("x-gigya-id_token", jwt);
+    public void actionHvacOn(double hvacTargetTemperature)
+            throws RenaultForbiddenException, RenaultNotImplementedException, RenaultActionException {
+        final String path = "/commerce/v1/accounts/" + kamereonaccountId + "/kamereon/kca/car-adapter/v1/cars/"
+                + config.vin + "/actions/hvac-start?country=" + getCountry(config);
+        postKamereonRequest(path,
+                "{\"data\":{\"type\":\"HvacStart\",\"attributes\":{\"action\":\"start\",\"targetTemperature\":\""
+                        + hvacTargetTemperature + "\"}}}");
+    }
 
+    public void actionChargeMode(ChargingMode mode)
+            throws RenaultForbiddenException, RenaultNotImplementedException, RenaultActionException {
         final String apiMode = ChargingMode.SCHEDULE_MODE.equals(mode) ? CHARGING_MODE_SCHEDULE : CHARGING_MODE_ALWAYS;
-        request.content(new StringContentProvider(
-                "{\"data\":{\"type\":\"ChargeMode\",\"attributes\":{\"action\":\"" + apiMode + "\"}}}", "utf-8"));
+        final String path = "/commerce/v1/accounts/" + kamereonaccountId + "/kamereon/kca/car-adapter/v1/cars/"
+                + config.vin + "/actions/charge-mode?country=" + getCountry(config);
+        postKamereonRequest(path,
+                "{\"data\":{\"type\":\"ChargeMode\",\"attributes\":{\"action\":\"" + apiMode + "\"}}}");
+    }
+
+    public void actionPause(PauseMode mode)
+            throws RenaultForbiddenException, RenaultNotImplementedException, RenaultActionException {
+        final String apiMode = PauseMode.PAUSE.equals(mode) ? "pause" : "resume";
+        final String path = "/commerce/v1/accounts/" + kamereonaccountId + "/kamereon/kcm/v1/vehicles/" + config.vin
+                + "/charge/pause-resume?country=" + getCountry(config);
```

Comment:
I did not know this existed.  It is not documented here: https://renault-api.readthedocs.io/en/latest/endpoints.html#action-endpoints.

Reply:
Hey,

you can find it here : https://renault-api.readthedocs.io/en/latest/endpoints/vehicle_kcm_actions.charge-pause-resume.html

The charge mode is not working on all model. For example I do have a Dacia Spring which is using the same Renault API, but there are some limitation. "Charging mode" is not compatible. The only thing I found to pause the charge is to use the above.

I did not migrate on OH 4 that's why I push on the 3.4 branch.

if you prefer I can try to do the same change on the OH 4 main branch.

Let me know what is you thoughs.

Vincent.

Reply:
@fifipil909  I think it would be easier for me to see your changes if we do the PR to main.  Maybe then a PR from main to 3.4 would be worth doing but it would need a bit of basic testing on 3.4.  I have migrated to 4-SNAPSHOTs so that is where I can test your changes on my wife's Zoe.  

For sure this is a useful feature.  We should document it and the charge_mode in the possible limitations of README.md if they do not work of some cars.  (however it is very hard to test on all car models etc.. but we should give the users the idea that some stuff might not work on their car).